### PR TITLE
fixed dupe descriptions

### PR DESCRIPTION
**Pull request by mairaw**
These should be the last remaining duplicate title/description pairs from SEO team's list.

### CODE REVIEW

File: docs/csharp/getting-started/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 title: Get Started - C# Guide
-description: Get Started with C#
+description: Find short, simple tutorials that let you quickly learn introductory C# concepts and how to write .NET Core applications.
```
Comment (rpetrusha) [inline docs/csharp/getting-started/index.md]: nit: better "that let you quickly learn...and write..."

File: docs/core/testing/unit-testing-with-dotnet-test.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
 ---
 title: Unit testing in .NET Core using dotnet test and xUnit
-description: Unit Testing in .NET Core using dotnet test
-keywords: .NET, .NET Core
+description: Learn unit test concepts in .NET Core through an interactive experience building a sample solution step-by-step using dotnet text and xUnit.
```
Comment (rpetrusha) [inline docs/core/testing/unit-testing-with-dotnet-test.md]: dotnet text --> dotnet test

**Review by rpetrusha — CHANGES_REQUESTED** [review]

**Comment by rpetrusha** [comment]
@mairaw, one nit and one minor change.

**Comment by mairaw** [comment]
Thanks @rpetrusha! I've addressed your comments.

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. This is ready to merge when you want.

**Pull request merged**